akamai/update_api.py

```diff
@@ -145,30 +125,28 @@ def activateVersion(version_number, env="STAGING"):
             print("Success! Version {} activated on {}.".format(version_number, env))
 
 
-def akamaiGet(url):
-    return s.get(urljoin(baseurl, url)).content
-
-def akamaiPost(url, body):
-    return s.post(urljoin(baseurl, url), json=body).content
 
-def akamaiPut(url, body):
-    return s.put(urljoin(baseurl, url), json=body).content
+def main():    
```

Comment:
Having the script in a `main` method makes the code look so much cleaner :smile: